### PR TITLE
Send to GA eligibility status from confirmation page

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

File: cla_public/apps/callmeback/views.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def request_callback():
 @callmeback.route('/result/confirmation')
 @redirect_if_no_session()
 def confirmation():
-    response = render_template('result/confirmation.html')
+    response = render_template('result/confirmation.html',
+        eligibility_status=session.eligibility_status)
```
Comment (andyhd) [inline cla_public/apps/callmeback/views.py]: This value is already available in the template via `{{ session.eligibility_status }}`, there's no need to pass it explicitly.

File: cla_public/apps/checker/session.py [hunk 2]
```diff
@@ -145,6 +145,20 @@ def aged_60_or_over(self):
...
+    @property
+    def eligibility_status(self):
+        status = 'unknown'
+        is_eligible = self.get('is_eligible', ELIGIBILITY_STATES.UNKNOWN)
+
+        if is_eligible == ELIGIBILITY_STATES.YES:
+            status = 'eligible'
+        elif is_eligible == ELIGIBILITY_STATES.NO:
+            status = 'ineligible'
+        elif is_eligible == ELIGIBILITY_STATES.UNKNOWN:
```
Comment (andyhd) [inline cla_public/apps/checker/session.py]: This branch is redundant

File: cla_public/apps/checker/session.py [hunk 3]
```diff
@@ -145,6 +145,20 @@ def aged_60_or_over(self):
     def callback_time(self):
         return self.get('CallMeBackForm_time')
 
+    @property
+    def eligibility_status(self):
+        status = 'unknown'
+        is_eligible = self.get('is_eligible', ELIGIBILITY_STATES.UNKNOWN)
```
Comment (andyhd) [inline cla_public/apps/checker/session.py]: The default value is not really needed here either.

File: cla_public/apps/callmeback/views.py (hunk 1 above)
Comment (tyom) [inline cla_public/apps/callmeback/views.py]: The reason it's passed explicitly is because the session is cleared on the following line.

Comment (andyhd) [inline cla_public/apps/callmeback/views.py]: The template is rendered before the session is cleared. The session var is available in the template.

File: cla_public/apps/checker/session.py (hunk 2 above)
Comment (tyom) [inline cla_public/apps/checker/session.py]: Ah, the initial `status` should be empty string. The idea is that the status only becomes unknown when the at least one form is submitted. Unfortunately this includes the details form, which I'm hoping can be fixed separately in near future.

File: cla_public/apps/checker/session.py (hunk 3 above)
Comment (tyom) [inline cla_public/apps/checker/session.py]: Currently `is_eligible` is set only after completing “About you” form. Therefore `eligibility_status` will throw when it's accessed from pages before to completing the “About you” form (i.e. going straight to “Call me back” page).

File: cla_public/apps/callmeback/views.py (hunk 1 above)
Comment (tyom) [inline cla_public/apps/callmeback/views.py]: You're right. Updating.

**Comment by andyhd** [comment]
:+1:

**Pull request merged**